src/extension/wps/src/test/java/au/org/emii/wps/gogoduck/parameter/SubsetParametersTest.java

```diff
@@ -36,10 +36,10 @@ public void testValidTime() {
 
     @Test
     public void testValidDepth() {
-        SubsetParameters sp = SubsetParameters.parse("TIME,2009-01-01T00:00:00.000Z,2009-12-25T23:04:00.000Z;LATITUDE,-33.433849,-32.150743;LONGITUDE,114.15197,115.741219;DEPTH,0.0,100.0");
+        SubsetParameters sp = SubsetParameters.parse("TIME,2009-01-01T00:00:00.000Z,2009-12-25T23:04:00.000Z;LATITUDE,-33.433849,-32.150743;LONGITUDE,114.15197,115.741219;DEPTH,43.567,100.0");
 
-        assertEquals(sp.getVerticalRange().first(), 0);
-        assertEquals(sp.getVerticalRange().last(), 100);
+        assertEquals((Object) sp.getVerticalRange().start, 43.567);
```

Comment:
Do you need to cast to Object here?  doesn't seem to be required above for getLonMin

Reply:
assertEquals(double, double) is deprecated - they want you to use the one with the delta